ArduPlane/quadplane.cpp

```diff
@@ -43,32 +43,32 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
     // @Path: ../libraries/AC_AttitudeControl/AC_PosControl.cpp
     AP_SUBGROUPPTR(pos_control, "P", 17, QuadPlane, AC_PosControl),
 
-    // @Param: VELZ_MAX
+    // @Param: PILOT_SPD_UP
```

Comment:
In coipter we use PILOT_SPEED_UP and PILOT_SPEED_DN but you probably already know this and it's because of the Q_ that you've shortened it?

Reply:
SPEED would fit, but matching copter which uses cm/s isn't actually a good thing right now